content/static/css/content.css

```diff
@@ -107,14 +107,22 @@ body {
   color:white;
 }
 
-.main { 
+.main {
   padding-top: 20px;
   width:100%;
   margin-right:0;
   margin-left:0;
   padding:0;
   }
-
+/*flatpages main content*/
+.maincontent  {
+  margin-top: 10px;
+  padding: 50px;
+  min-height:550px;
```

Comment:
some nitpicking but not sure if you guys want spaces or no spaces after colons, but lets make this consistent